openmetadata-ui/src/main/resources/ui/src/utils/RouterUtils.ts

```diff
@@ -130,7 +130,7 @@ export const getExplorePathWithInitFilters = (
     .replace(PLACEHOLDER_ROUTE_TAB, tab);
 
   return filter
-    ? `${path}?${initialFilterQS}=${encodeURIComponent(filter)}`
+    ? `${path}?${searchFilterQS}=${encodeURIComponent(filter)}`
```

Comment:
@ShaileshParmar11 , Any reason why we are changing this to serach filters?

Also can you check if we change the tab does tag filter getting applied or not?

Reply:
I have updated the code and brought back the `initialFilterQS `, and for the tab switch filters are getting reset! to keep it applied we need to deep dive into the code to apply the functionality we will do it in the next iteration!